packages/devtools/src/datamodel.ts

```diff
@@ -252,6 +297,42 @@ export class ModelManager {
     }, {} as Record<string, Array<string>>)
   }
 
+  // High-level
+
+  async create<T extends keyof CreateContentType, Content = CreateContentType[T]>(
+    type: T,
+    alias: string,
+    content: Content
+  ): Promise<ManagedID> {
+    switch (type) {
+      case 'schema':
+        return await this.createSchema(alias, content as any)
+      case 'definition':
+        return await this.createDefinition(alias, content as any)
+      case 'tile':
+        return await this.createTile(alias, content as any)
+      default:
+        throw new Error(`Unsupported type: ${type as string}`)
+    }
+  }
+
+  async usePublished<T extends keyof UsePublishedIDType, ID = UsePublishedIDType[T]>(
+    type: T,
+    alias: string,
+    id: ID
+  ): Promise<ManagedID> {
+    switch (type) {
+      case 'schema':
+        return await this.usePublishedSchema(alias, id as any)
```

Comment:
I think for these 3 (usePublishedSchema, usePublishedDefinition & usePublishedTile) we should have a type associated with them. I really don't like typing as any for something we're returning. Even a flexible type would be awesome.

Reply:
These types cast are internal to the method call because TS doesn't seem to properly understand the method arguments, but these should have proper types.